parcels/fieldset.py

```diff
@@ -194,21 +165,48 @@ def add_constant_field(self, name: str, value: float, mesh: Mesh = "flat"):
                correction for zonal velocity U near the poles.
             2. flat: No conversion, lat/lon are assumed to be in m.
         """
-        self.add_field(Field(name, value, lon=0, lat=0, mesh=mesh))
+        time = 0.0
+        values = np.zeros((1, 1, 1, 1), dtype=np.float32) + value
+        data = xr.DataArray(
```

Comment:
```suggestion
        values = np.full((1, 1, 1, 1), value)
        data = xr.DataArray(
```
And update the docstring for `value : float` to remove 32 bit requirement?

Reply:
Resolved in https://github.com/OceanParcels/Parcels/pull/1946/commits/9d3a7e28907ec56428f9e9efffeee35941a3890c